### DISCUSSION TITLE
Why is my local Photon getting different results?

### DISCUSSION THREAD

**Discussion by gpofcher** [komoot/photon#628] [Q&A] [answered]
https://photon.komoot.io/api?q=224%20main%20st%20brattleboro&lon=-72.55993603134883&lat=42.8552926 gets exactly what I want: 
`{"features":[{"geometry":{"coordinates":[-72.55993603134883,42.8552926],"type":"Point"},"type":"Feature","properties":{"osm_id":320467897,"extent":[-72.5601733,42.8554619,-72.5596989,42.8551253],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"amenity","housenumber":"224","street":"Main Street","osm_value":"library","name":"Brooks Memorial Library","state":"Vermont"}}],"type":"FeatureCollection"}`
same query locally has a bunch of false positives and not the correct result:
`{"features":[{"geometry":{"coordinates":[-72.55954555271691,42.854506650000005],"type":"Point"},"type":"Feature","properties":{"osm_id":210070659,"extent":[-72.5598674,42.8546946,-72.5592237,42.8542828],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"amenity","housenumber":"204","street":"Main Street","osm_value":"post_office","name":"Brattleboro Post Office","state":"Vermont"}},{"geometry":{"coordinates":[-72.5595567710289,42.8548841],"type":"Point"},"type":"Feature","properties":{"osm_id":284347545,"extent":[-72.5597081,42.8549809,-72.5594043,42.8547781],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"amenity","housenumber":"210","street":"Main Street","osm_value":"arts_centre","name":"Brattleboro Stone Church","state":"Vermont"}},{"geometry":{"coordinates":[-72.5595567710289,42.8548841],"type":"Point"},"type":"Feature","properties":{"osm_id":284347545,"extent":[-72.5597081,42.8549809,-72.5594043,42.8547781],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"leisure","housenumber":"210","street":"Main Street","osm_value":"dance","name":"Brattleboro Stone Church","state":"Vermont"}},{"geometry":{"coordinates":[-72.55821457219098,42.8504406],"type":"Point"},"type":"Feature","properties":{"osm_id":569997219,"extent":[-72.5586221,42.8506535,-72.5576955,42.8502483],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"shop","housenumber":"2","street":"Main Street","osm_value":"supermarket","name":"Brattleboro Food Co-Op","state":"Vermont"}},{"geometry":{"coordinates":[-72.5611840353866,42.8554406],"type":"Point"},"type":"Feature","properties":{"osm_id":293902711,"extent":[-72.5614504,42.8556396,-72.5609177,42.8551689],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"office","housenumber":"230","street":"Main Street","osm_value":"government","name":"Brattleboro Municipal Center","state":"Vermont"}},{"geometry":{"coordinates":[-72.5590416654382,42.855459350000004],"type":"Point"},"type":"Feature","properties":{"osm_id":615265560,"extent":[-72.5592598,42.8556136,-72.5588445,42.8553103],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301-8690","county":"Windham County","type":"house","osm_type":"W","osm_key":"building","street":"Main Street","osm_value":"yes","name":"Brattleboro Savings & Loan","state":"Vermont"}},{"geometry":{"coordinates":[-72.55903308638207,42.85360565],"type":"Point"},"type":"Feature","properties":{"osm_id":320467891,"extent":[-72.5592622,42.8536861,-72.5587786,42.8535269],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301-8690","county":"Windham County","type":"house","osm_type":"W","osm_key":"building","street":"Main Street","osm_value":"yes","name":"Brattleboro Area Chamber of Commerce","state":"Vermont"}},{"geometry":{"coordinates":[-72.5589789,42.8542486],"type":"Point"},"type":"Feature","properties":{"osm_id":356557345,"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301-8690","county":"Windham County","type":"house","osm_type":"N","osm_key":"historic","street":"Main Street","osm_value":"district","name":"Brattleboro Downtown Historic District","state":"Vermont"}},{"geometry":{"coordinates":[-72.5580022,42.8522173],"type":"Point"},"type":"Feature","properties":{"osm_id":209226074,"extent":[-72.558517,42.853247,-72.5575248,42.8510753],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301-8690","county":"Windham County","type":"street","osm_type":"W","osm_key":"highway","osm_value":"primary","name":"Main Street","state":"Vermont"}},{"geometry":{"coordinates":[-72.586228,42.857633],"type":"Point"},"type":"Feature","properties":{"osm_id":19734157,"extent":[-72.586776,42.859296,-72.58595,42.856665],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"street","osm_type":"W","osm_key":"highway","osm_value":"residential","name":"Macintosh Lane","state":"Vermont"}},{"geometry":{"coordinates":[-72.55852635866323,42.85463775],"type":"Point"},"type":"Feature","properties":{"osm_id":283428562,"extent":[-72.5588414,42.8547902,-72.5582113,42.8544853],"country":"United States","city":"Brattleboro","countrycode":"US","postcode":"05301","county":"Windham County","type":"house","osm_type":"W","osm_key":"leisure","housenumber":"207","street":"Main Street","osm_value":"sports_centre","name":"Gibson-Aiken Center","state":"Vermont"}},{"geometry":{"coordinates":[-72.3090092090458,42.5829261],"type":"Point"},"type":"Feature","properties":{"osm_id":214935141,"extent":[-72.3090817,42.5829924,-72.3089367,42.5828911],"country":"United States","city":"Orange","countrycode":"US","postcode":"01364-9998","county":"Franklin County","type":"house","osm_type":"W","osm_key":"building","housenumber":"224","street":"South Main Street","osm_value":"yes","state":"Massachusetts"}},{"geometry":{"coordinates":[-72.6085356475942,42.49682345],"type":"Point"},"type":"Feature","properties":{"osm_id":212602223,"extent":[-72.6086181,42.4968942,-72.60847,42.4966626],"country":"United States","city":"Deerfield","countrycode":"US","postcode":"01373","county":"Franklin County","type":"house","osm_type":"W","osm_key":"building","housenumber":"224","street":"North Main Street","osm_value":"yes","state":"Massachusetts"}},{"geometry":{"coordinates":[-72.7037328,43.3955295],"type":"Point"},"type":"Feature","properties":{"osm_id":9084185583,"country":"United States","city":"Ludlow","countrycode":"US","postcode":"05149","county":"Windsor County","type":"house","osm_type":"N","osm_key":"building","housenumber":"224","street":"Main Street","district":"Village of Ludlow","osm_value":"commercial","state":"Vermont"}},{"geometry":{"coordinates":[-72.70384425262336,43.3956947],"type":"Point"},"type":"Feature","properties":{"osm_id":374639973,"extent":[-72.7039201,43.3957697,-72.7037684,43.3956196],"country":"United States","city":"Ludlow","countrycode":"US","postcode":"05149","county":"Windsor County","type":"house","osm_type":"W","osm_key":"amenity","housenumber":"224","street":"Main Street","district":"Village of Ludlow","osm_value":"fuel","name":"Sunoco","state":"Vermont"}}],"type":"FeatureCollection"}`
Is there a problem with my data? Do I need to structure my query better?

**Comment by lonvia** [comment]
Hard to say unless you tell us about your local instance. Which photon, which data, what did you do to set it up etc.

**Reply by gpofcher** [reply]
Photon-0.3.5

I followed the installation instructions from the readme as closely as I could, although I ran into some trouble with the decompression part—first Norton blocked pbzip2, then I ran out of space decompressing the tar and had to resume partway through.

I'm running Windows 10 21H2.

**Comment by gpofcher** [comment, marked as answer]
Update: This morning I discovered the US Census geocoder at https://geocoding.geo.census.gov/geocoder/locations/addressbatch?form which does what I need more easily.
